source/lib/predicates/number.ts

```diff
@@ -131,4 +131,14 @@ export class NumberPredicate extends Predicate<number> {
 			validator: value => value < 0
 		});
 	}
+
+	/**
+	 * Test a number to be an integer or infinite
+	 */
+	get integerOrInfinite() {
+		return this.addValidator({
+			message: (value, label) => `Expected ${label} to be integer or infinity, got ${value}`,
```

Comment:
Maybe?

> Expected ${label} to be an integer or infinite, got ${value}

Reply:
Yes, I think this makes sense so it's consistent with naming of the function and the doc commen :slightly_smiling_face: